src/components/browse-by/browse-by.js

```diff
@@ -0,0 +1,89 @@
+import SETTINGS from "../../global/settings";
+
+const browseBy = () => {
+	if (document.querySelector(".js-browseby")) {
+		const browseBys = document.querySelectorAll(".js-browseby");
+		const mobileBreakpoint = SETTINGS.mobileBreakpoint;
```

Comment:
since `SETTINGS.mobileBreakpiont` is already a constant I would just reference that everywhere instead of duplicating into another named variable -- just adds more places things can go wrong - less code is best code